### PR TITLE
remove old CORS allow list entries

### PR DESCRIPTION
**Pull request by camallen**
Extracted from #3345 

Remove local dev work ip based entries in allow list, folks switch to `local.zooniverse.org` or `myapp.zooniverse.org` with local DNS (/etc/hosts) overrides.

Remove old CFE apps that are already redirecting to www.zooniverse.org

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: kubernetes/deployment-production.tmpl [hunk 1]
```diff
@@ -171,7 +171,7 @@ spec:
                   name: panoptes-common-env-vars
                   key: AWS_SECRET_ACCESS_KEY
             - name: CORS_ORIGINS_REGEX
-              value: '^https?:\/\/((www|staging)\.antislaverymanuscripts\.org|(www|staging)?\.?heirtagger\.ox\.ac\.uk|(www|lab|learn)\.wildcamgorongosa\.org|www\.diagnosislondon\.org|anno\.tate\.org\.uk|(www|relaunch|field-book)\.notesfromnature\.org|[a-z0-9-]+\.oldweather\.org|www\.scribesofthecairogeniza\.org|([a-z0-9-]+\.)?zooniverse\.org|[a-z0-9-]+\.(pfe-)?preview\.zooniverse\.org)(:\d+)?$'
+              value: '^https?:\/\/((www|staging)\.antislaverymanuscripts\.org|(www|staging)?\.?heirtagger\.ox\.ac\.uk|(www|relaunch|field-book)\.notesfromnature\.org|[a-z0-9-]+\.oldweather\.org|www\.scribesofthecairogeniza\.org|([a-z0-9-]+\.)?zooniverse\.org|[a-z0-9-]+\.(pfe-)?preview\.zooniverse\.org)(:\d+)?$'
```
Comment (camallen) [inline kubernetes/deployment-production.tmpl:174]: Can we remove `*.oldweather.org` entries? Or be more explicit in the allow list? 

Can any other entries be removed?

Comment (wgranger) [reply kubernetes/deployment-production.tmpl:174]: It looks like oldweather.org is only an archive page. Doesn't look like it needs Panoptes.

We still need ASM and Scribes of the Cairo Geniza.

Comment (wgranger) [reply kubernetes/deployment-production.tmpl:174]: There's also "classic.oldweather.org," which gives a ngix error, and "whaling.oldweather.org," which is also an archived page

Comment (camallen) [reply kubernetes/deployment-production.tmpl:174]: thanks for doing the leg work will, i just tested https://classic.oldweather.org/ and it uses oauth redirection flow to login so doesn't need cors access. I'll remove it and others :)

Comment (camallen) [reply kubernetes/deployment-production.tmpl:174]:
```suggestion
              value: '^https?:\/\/((www|staging)\.antislaverymanuscripts\.org|(www|staging)?\.?heirtagger\.ox\.ac\.uk|(www|relaunch|field-book)\.notesfromnature\.org|www\.scribesofthecairogeniza\.org|([a-z0-9-]+\.)?zooniverse\.org|[a-z0-9-]+\.(pfe-)?preview\.zooniverse\.org)(:\d+)?$'
```

**Comment by adammcmaster** [comment]
Just noting that this will conflict with #3350 since that refactors the env vars. Rather than merging this it might be easier to just add a new commit to the other PR.

**Pull request merged**